crud.py

```diff
@@ -0,0 +1,65 @@
+from sqlalchemy.orm import Session
+
+from db import models
+import schemas
+
+
+def get_all_authors(db: Session):
```

Comment:
also if you use annotations, use them for return types as well

Reply:
annotations added